### ISSUE TITLE
Doesn’t parse ‘Return value’ tag

### ISSUE THREAD

**Issue by pwithnall** [MathieuDuponchelle/hotdoc#29]
If I use
```
 * Return value: (transfer full): a new #WblSchema; unref with g_object_unref()
```
instead of
```
 * Returns: (transfer full): a new #WblSchema; unref with g_object_unref()
```

hotdoc doesn’t understand the line, and treats it as part of the documentation body.

**Comment by MathieuDuponchelle** [comment]
I'm not aware of this being supported by gtk-doc, can you point me to projects using that ? Also unrelated but you should replace "unref with g_object_unref" with the "unref-func" annotation, which will get properly discarded when building documentation for python and js :)

**Comment by pwithnall** [comment]
> I'm not aware of this being supported by gtk-doc, can you point me to projects using that ?

It is the syntax originally supported by gtk-doc, before `Returns` was introduced.

https://wiki.gnome.org/DocumentationProject/GtkDoc#line-195

> you should replace "unref with g_object_unref" with the "unref-func" annotation, which will get properly discarded when building documentation for python and js :)

Indeed, although it might be an idea for hotdoc to support extracting phrases like that, since a huge amount of documentation which was written before GIR still says ‘free with g_free()’ or ‘unref with g_object_unref()’ everywhere.

**Comment by MathieuDuponchelle** [comment]
Hm OK, not sure I want to support that legacy syntax tbh, might have been deprecated long enough  now ? (note it would be a one-liner to support, no laziness involved :P)

As for the unref functions, hotdoc has no plans to become a natural language parser, there are annotations for that purpose :)

**Comment by MathieuDuponchelle** [comment]
Actually not deprecated, grmph

**Comment by MathieuDuponchelle** [comment]
Weird I hadn't seen it in the gtk sources when generating doc btw, I must have been content seeing it built and looked reasonably well.

**Comment by MathieuDuponchelle** [comment]
fixed by 5591bf3899a3b24a8015290b96b08daa4124cd30 , thanks !
